.travis.yml

```diff
@@ -65,6 +65,26 @@ matrix:
       go: 1.15.x
       script:
         - make build-backend
+    - name: "Verify api-server, persistenceagent, metadata-writer, and scheduledworkflow docker build"
+      services: docker
+      env:
+        - DOCKER_REGISTRY=fake_registry
+        - DIFF_DETECTED_ERR_CODE=169
+      before_script:
+        # No diff detected:                   terminate job with success i.e. travis_terminate 0
+        # Script throws unexpected error:     fail job with error code
+        # Diff detected:                      continue/run this job
+        - |
+          ./scripts/check_diff.sh
+          EXIT_CODE=$?
+          if [ $EXIT_CODE -eq 0 ]; then
+            travis_terminate 0
```

Comment:
Thanks @evan-hataishi, maybe have something print or echo to the console when the build is skipped? Some people may not know why it exit with 0.

Reply:
Sounds good. 

I still have a few modifications left. I totally forgot this triggers the build every time I push. I was amending my commits, so I could run travis on my forked repo to see if everything was working the way I intended it to (I think there's still a couple issues). I'm not sure if there's a way to avoid triggering the build on this PR until it's ready... There's probably a better way to test travis than this :)

Reply:
From your end, you can add this to your commit message
https://docs.travis-ci.com/user/customizing-the-build/#skipping-a-build

Reply:
Ahh I see, but I still want it to build under my forked repo's travis (https://travis-ci.com/github/evan-hataishi/kfp-tekton). I don't think there's a way to do avoid it unless I close this PR, which might not actually be a bad option...

Reply:
Ohh, actually nevermind. I can add the skip for now and just run the build manually!

Reply:
Nevermind again :( I can't even trigger the a manual build, since the skip is in the commit message.

Reply:
Nevermind...nevermind. I can add a custom commit message when triggering the build manually, yay.